include/Gaffer/PlugAlgo.h

```diff
@@ -71,6 +71,12 @@ bool canPromote( const Plug *plug, const Plug *parent = NULL );
 /// related to plug layouts are copied to the external
 /// plug - this can be controlled with the `excludeMetadata`
 /// argument.
+///
+/// > Note : When promoting to a Box, this function examines
+/// > the value of the "nodule:type" metadata, and automatically
+/// > creates an intermediate BoxIO node if the plug has a
+/// > nodule. This makes the exact effects of the function dependent
+/// > on whether or not the UI metadata has been loaded.
```

Comment:
I edited your (7) in place with Daniel's idea from the meeting. I thought it was easier to understand if we had them all written in place...